sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java

```diff
@@ -150,6 +150,44 @@ public EditContactsSteps(
               .isEqualTo(collectedContact.getDescriptionOfHowContactTookPlace());
           softly.assertAll();
         });
+
+    Then(
+        "I check the linked contact information is correctly displayed",
+        () -> {
+          String contactId = webDriverHelpers.getValueFromTableRowUsingTheHeader("Contact ID", 1);
+          String contactDisease =
+              (webDriverHelpers.getValueFromTableRowUsingTheHeader("Disease", 1).equals("COVID-19"))
+                  ? "CORONAVIRUS"
+                  : "Not expected string!";
+          String contactClassification =
+              (webDriverHelpers
+                      .getValueFromTableRowUsingTheHeader("Contact classification", 1)
+                      .equals("Unconfirmed contact"))
+                  ? "UNCONFIRMED"
+                  : "Not expected string!";
+          String firstName =
+              webDriverHelpers.getValueFromTableRowUsingTheHeader(
+                  "First name of contact person", 1);
+          String lastName =
+              webDriverHelpers.getValueFromTableRowUsingTheHeader("Last name of contact person", 1);
+
+          softly
+              .assertThat(apiState.getCreatedContact().getUuid().substring(0, 6))
+              .isEqualToIgnoringCase(contactId);
```

Comment:
Same as above: please document why 0 to 6 index substring of apiState.getCreatedContact().getUuid() equals contactId.